programs/ziti-edge-tunnel/CMakeLists.txt

```diff
@@ -14,6 +15,16 @@ if(CMAKE_SYSTEM_NAME STREQUAL Windows)
     set(NETIF_DRIVER_SOURCE netif_driver/windows/tun.c netif_driver/windows/tun.h)
 endif()
 
+if (MSVC)
+    message("using visual studio")
+    FetchContent_Declare(win-c
+            GIT_REPOSITORY https://github.com/ekoby/win-c.git
```

Comment:
why `https://github.com/ekoby/win-c.git`  vs /netfoundry?

Reply:
argh, good catch, I was trying it our first and didn't want to pollute org if it didn't workout